README.md

```diff
@@ -18,6 +18,14 @@ Thanks to all of AMS contributors.
 
 AMS was created under the AMS LDRD-SI project (22-SI-004).
 
+## Citation
+
+If you use this software, please cite it as below:
+
+- Bhatia, Harsh, Patki, Tapasya A., Brink, Stephanie, Pottier, Loic E., Stitt, Thomas M., Parasyris, Konstantinos, Milroy, Daniel J., Laney, Daniel E., Blake, Robert C., Yeom, Jae-Seung, Bremer, Peer-Timo, and Doutriaux, Charles. Autonomous MultiScale Library. Computer Software. https://github.com/LLNL/AMS. USDOE National Nuclear Security Administration (NNSA). 01 May. 2023. Web. doi:10.11578/dc.20230721.1.
```

Comment:
```suggestion
- Bhatia, Harsh, Patki, Tapasya A., Brink, Stephanie, Pottier, Loïc, Stitt, Thomas M., Parasyris, Konstantinos, Milroy, Daniel J., Laney, Daniel E., Blake, Robert C., Yeom, Jae-Seung, Bremer, Peer-Timo, and Doutriaux, Charles. Autonomous MultiScale Library. Computer Software. https://github.com/LLNL/AMS. US DOE National Nuclear Security Administration (NNSA). 01 May. 2023. Web. doi:10.11578/dc.20230721.1.
```

Reply:
@koparasy: I don't know if that's important but the order in the CITATION.cff and the README do not match.